components/ee/agent-smith/pkg/agent/agent.go

```diff
@@ -701,7 +701,10 @@ func (agent *Smith) handleExecveEvent(execve Execve) func() (*InfringingWorkspac
 
 			for _, b := range bl.Binaries {
 				if strings.Contains(execve.Filename, b) || strings.Contains(strings.Join(execve.Argv, "|"), b) {
-					infr := Infringement{Description: fmt.Sprintf("user ran %s blacklisted command: %s", s, execve.Filename), Kind: GradeKind(InfringementExecBlacklistedCmd, s)}
+					infr := Infringement{
+						Description: fmt.Sprintf("user ran %s blacklisted command: %s %v", s, execve.Filename, execve.Envp),
```

Comment:
Unfortunately, I don't think `execve.Envp` is ever set:

https://github.com/gitpod-io/gitpod/blob/f646642c05bbd29f520b0df5e9f5fc40bc160318/components/ee/agent-smith/pkg/agent/agent.go#L629-L633

Reply:
fixed